app/components/agenda/agendaitem/agendaitem-case/agendaitem-titles-edit.hbs

```diff
@@ -45,6 +45,18 @@
       />
     </div>
   </div>
+  <div class="auk-u-mb-4">
+    <div class="auk-form-group">
+      <Auk::Label id="private-comment-title" class="auk-u-icon-left"><Auk::Icon @name="lock-closed" @size="small" /> {{t "private-comment-title"}}</Auk::Label>
+      <Textarea
+        data-test-agendaitem-titles-edit-private-explanation={{true}}
```

Comment:
You don't really have to add test selectors if you're not using them in this PR.
It will just result in unused and untracked selectors (selector files in cypress).
Opt to remove any new ones from this PR.

Reply:
Updated the agenda.spec.js test to check the private comment alongside the normal comment, so left this in

Reply:
I reviewed before any test was changed so this is good now.